src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/ImportProcess.java

```diff
@@ -280,8 +282,8 @@ else if (coverage != null) {
         else if (coverage != null)
         {
         	try {
-        		final File directory = catalog.getResourceLoader().findOrCreateDirectory("data", workspace, store);
-        		final File file = File.createTempFile(store, ".tif", directory);
+        		final Resource directory = catalog.getResourceLoader().get(Paths.path("data", workspace, store));
+        		final File file = File.createTempFile(store, ".tif", directory.dir());
```

Comment:
Aaargh, more tabs! I know these are not your fault. @NielsCharlier please remove all tabs in any method you touch. You can select the method in Eclipse and reformat it to avoid changing the entire file. Making the world a better place, one tab at a time.